SofaKernel/modules/SofaBaseLinearSolver/src/SofaBaseLinearSolver/CGLinearSolver.inl

```diff
@@ -108,41 +96,44 @@ void CGLinearSolver<TMatrix,TVector>::solve(Matrix& M, Vector& x, Vector& b)
     simulation::Visitor::printNode("VectorAllocation");
 #endif
 
+    /// Allocate the required vectors for the iterative resolution
     const core::ExecParams* params = core::execparams::defaultInstance();
-    typename Inherit::TempVectorContainer vtmp(this, params, M, x, b);
-    Vector& p = *vtmp.createTempVector();
-    Vector& q = *vtmp.createTempVector();
-    Vector& r = *vtmp.createTempVector();
+    typename Inherit::TempVectorContainer vtmp(this, params, A, x, b);
+    Vector& p = *vtmp.createTempVector(); // orthogonal directions
+    Vector& q = *vtmp.createTempVector(); // temporary vector computing A*p
+    Vector& r = *vtmp.createTempVector(); // residual
 
-    const bool verbose  = f_verbose.getValue();
+    const bool printLog  = this->f_printLog.getValue();
     double rho, rho_1=0, alpha, beta;
 
 
-    msg_info_when(verbose) << "b = " << b ;
+    msg_info_when(printLog) << "b = " << b ;
```

Comment:
I think `msg_info()` is called only if `printLog` is true. So `msg_info_when(printLog)` is redundant. Prefer `msg_info() << "b = " << b ;`